package.yaml

```diff
@@ -0,0 +1 @@
+name: dyp_a01
```

Comment:
Alternatively you can change the README.md title to `dyp_a01` (when it's taken from the README, it's automatically lower-cased, so `DYP_A01` would also work).